bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1326,6 +1326,12 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
          lnd_tuning_mode="clm5_0_GSWP3v1" use_init_interp=".true."
 >lnd/clm2/initdata_esmf/ctsm5.3/clmi.interp_from.I1850Clm50BgcCrop-ciso.1366-01-01.0.9x1.25_gx1v7_simyr1850_c240223.nc
 </finidat>
+<finidat hgrid="0.9x1.25" maxpft="79" mask="tx2_3v2" use_cn=".true." use_cndv=".false." use_fates=".false."
+         ic_ymd="18500101" sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
+         ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".false."
+         phys="clm6_0" use_init_interp=".true."
```

Comment:
This is a new file that was created by interpolating from the original right? It should then have

use_init_interp either removed or set to .false. Same with the one just above.

I think it's also true of this one:

lnd/clm2/initdata_esmf/ctsm5.4/clmi.f19_twiceinterp_from.I1850Clm50BgcCrop-ciso.1366-01-01.0.9x1.25_gx1v7_simyr1850_c251030.nc